src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Statistics/TableBuilderControllerIntegrationTests.cs

```diff
@@ -44,15 +45,18 @@ public class TableBuilderControllerIntegrationTests(TestApplicationFactory testA
             }
         };
 
+        private static readonly ObservationQueryContext ObservationQueryContext =
+            FullTableQueryRequest.AsObservationQueryContext();
+
         private readonly TableBuilderResultViewModel _tableBuilderResults = new()
         {
             SubjectMeta = new SubjectResultMetaViewModel
             {
-                TimePeriodRange = new List<TimePeriodMetaViewModel>
-                {
-                    new(2020, AcademicYear),
-                    new(2021, AcademicYear),
-                }
+                TimePeriodRange =
+                [
+                    new TimePeriodMetaViewModel(2020, AcademicYear),
```

Comment:
Does this new C# "feature" mean you have to define the types explicitly again? Normally the `TimePeriodMetaViewModel` type would be inferred for each item in the collection.

Reply:
It doesn't but Rider was recommending showing the type name again - I assume the rationale here is that since we don't see the type in `new List<TimePeriodMetaViewModel>` it prefers to explicitly say the type when creating elements.

I'm not really fussed either way - I think my preference would be to have the type stated somewhere, but it's not too hard to hover over `TimePeriodRange` to see the type.